src/transformers/modeling_utils.py

```diff
@@ -4098,12 +3726,15 @@ def _fix_key(key):
                     target_dtype = torch.float32
 
                 if param.device == torch.device("meta"):
-                    if not (is_quantized):
-                        set_module_tensor_to_device(model, key, "cpu", torch.empty(*param.size(), dtype=target_dtype))
+                    value = torch.empty(*param.size(), dtype=target_dtype)
+                    if getattr(
```

Comment:
Missed a guard clause here to check `if hf_quantizer is not None and getattr(`

Reply:
Thanks ! This should be fixed here: https://github.com/huggingface/transformers/pull/28804 no?